doc/pseudocode.md

```diff
@@ -0,0 +1,152 @@
+public class SQLQueryBuilder
+{
```

Comment:
why you make this file extension .md? as you write c#, you should use .cs extension